packages/hardhat/contracts/FundingVaultFactory.sol

```diff
@@ -37,14 +39,24 @@ contract FundingVaultFactory{
     error FundingVaultFactory__deadlineCannotBeInThePast();
     error FundingVaultFactory__MinFundingAmountCanNotBeZero();
     error FundingVault__TokenTransferFailed();
+    error FundingVaultFactory__InvalidIndex();
 
+    struct Vault{
```

Comment:
Add a space before "{".

Check the whole code for this styling issue.

Reply:
Fixed